tests/boundary_based/test_sample_boundary.py

```diff
@@ -0,0 +1,45 @@
+# Copyright IRT Antoine de Saint Exupéry et Université Paul Sabatier Toulouse III - All
+# rights reserved. DEEL is a research program operated by IVADO, IRT Saint Exupéry,
+# CRIAQ and ANITI - https://www.deel.ai/
+# =====================================================================================
+import tensorflow as tf
+from tensorflow.keras.layers import Input, Dense
+from tensorflow.keras.models import Sequential
+
+from deel.influenciae.boundary_based import SampleBoundaryCalculator
+
+
+def test_compute_influence_shape():
```

Comment:
I think we should not have tests with constant values and without explaining how we get the `expected_results` from those constants. It makes the test really unclear on how it challenge the results

Reply:
Hope we do not forget to change it in the next PR